clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -1166,24 +1172,19 @@ FunctionVariableConstraint::FunctionVariableConstraint(
       DidConvert = true;
     } else {
       if (!Ext->isOriginallyChecked()) {
-        Ext->constrainToWild(CS, VOID_TYPE_REASON);
+        Ext->constrainToWild(CS, ReasonLoc(VOID_TYPE_REASON,PSL));
```

Comment:
It would be useful to add a PSL to the base class `ConstraintVariable`. Is it worth trying to do that for this PR or should I write it up as a follow-up enhancement?